cmd/ctr/commands/run/run_unix.go

```diff
@@ -226,6 +231,10 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 			opts = append(opts, oci.WithCPUCFS(quota, period))
 		}
 
+		if shares := context.Int("cpu-shares"); shares > 0 {
+			opts = append(opts, oci.WithCPUShares(uint64(shares)))
```

Comment:
`WithCPUShares` needs a definition in `oci/spec_opts_unix.go` or this logic moved to a `_linux.go` file

Reply:
https://github.com/containerd/containerd/blob/main/oci/spec_opts_linux.go#L88

Reply:
```
2021-08-09T16:39:21.8708090Z level=warning msg="[runner] Can't run linter unused: buildir: analysis skipped: errors in package: [/Users/runner/work/containerd/containerd/cmd/ctr/commands/run/run_unix.go:240:28: WithCPUShares not declared by package oci]"
```
Doesn't add, up wrong line number and the function is there..

Reply:
It should be fine now, you probably had an old diff, i just pushed the changes.